### PR TITLE
Added dynamic configuration file for paella and xapi plugin params

### PR DESCRIPTION
**Pull request by dpeite**
An example of xapi params to include in parameters.yml:

```yaml
pumukit_paella:
        endpoint: 'https://localhost:90/data/xAPI/'
        auth: 'b65a1ab5225a1068168959fa01b6bb18644bdbef:ece3becf84db9d9c84b52ab720438c4ccddd7606'
```

### CODE REVIEW

File: Controller/ConfController.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    */
+    public function confAction(Request $request)
+    {
+        $endpoint = $this->getParameter('pumukitpaella.xapi_endpoint');
+        $auth =  $this->getParameter('pumukitpaella.xapi_auth');
+        $accessControlClass = $this->getParameter('pumukitpaella.access_control_class');
+
+        $jsonData = $this->renderView('PumukitBasePlayerBundle:Conf:conf.json.twig', array('xapi_endpoint' => $endpoint, 'xapi_auth' => $auth, 'access_control_class' => $accessControlClass));
+
+        return new Response($jsonData, 200, array('Content-Type'=>'application/json'));
```
Comment (rubenrua) [inline Controller/ConfController.php]: use `JsonResponse`

```diff
-return new Response($jsonData, 200, array('Content-Type'=>'application/json'));
+return new JsonResponse($jsonData);
```

https://symfony.com/doc/current/components/http_foundation.html#creating-a-json-response

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -25,6 +25,18 @@ public function getConfigTreeBuilder()
...
             ->end()
+            ->scalarNode('xapi_endpoint')
+              ->defaultValue(null)
+              ->info('LRS endpoint for xAPI')
+            ->end()
+            ->scalarNode('xapi_auth')
+              ->defaultValue(null)
+              ->info('LRS auth token for xAPI')
+            ->end()
+            ->scalarNode('access_control_class')
```
Comment (rubenrua) [inline DependencyInjection/Configuration.php]: No use a `scalarNode`. Use an `enumNode` with values

https://symfony.com/doc/current/components/config/definition.html#enum-nodes

File: Resources/config/services.xml [hunk 3]
```diff
@@ -4,6 +4,11 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+
+  <parameters>
```
Comment (rubenrua) [inline Resources/config/services.xml]: Why?

File: Resources/public/paella/player/config/config.json [hunk 4]
```diff
@@ -131,7 +131,8 @@
 
             "//****": "User Tracking Saver Plugins",
             "es.upv.paella.usertracking.elasticsearchSaverPlugin": { "enabled": false, "url": "http://my.elastic.server"},
-            "es.upv.paella.usertracking.GoogleAnalyticsSaverPlugIn": { "enabled": false, "trackingID": "UA-XXXXXXXX-Y" }
+            "es.upv.paella.usertracking.GoogleAnalyticsSaverPlugIn": { "enabled": false, "trackingID": "UA-XXXXXXXX-Y" },
+						"es.teltek.paella.usertracking.xAPISaverPlugin": {"enabled": true, "endpoint":"https://localhost:90/data/xAPI/", "auth":"b65a1ab5225a1068168959fa01b6bb18644bdbef:ece3becf84db9d9c84b52ab720438c4ccddd7606"}
```
Comment (rubenrua) [inline Resources/public/paella/player/config/config.json]: Review indentation.

File: Services/PaellaDataService.php [hunk 5]
```diff
@@ -51,21 +51,21 @@ public function getPaellaPlaylistData(Series $series, Request $request, $criteri
...
-                'pumukit_playlistplayer_paellaindex',
-                array(
-                    'playlistId' => $series->getId(),
-                    'videoId' => $mmobj->getId(),
-                    'videoPos' => $pos,
-                    'autostart' => 'true',
-                ),
-                true  //Makes the url absolute.
-            );
+        'pumukit_playlistplayer_paellaindex',
```
Comment (rubenrua) [inline Services/PaellaDataService.php]: Review indentation

File: DependencyInjection/Configuration.php (hunk 2 above)
Comment (dpeite) [reply DependencyInjection/Configuration.php]: Paella force a default access_control_class if param not exist or it's empty

**Pull request merged**